rna/README.md

```diff
@@ -72,4 +72,22 @@ The `samples` table would then contain, for each sample:
 - [other samtools output files]
 - featureCounts_files (Array of STRINGS: paths to featureCounts output files)
 
-All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.
\ No newline at end of file
+All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.
+
+Additionally, you can run Picard tools to produce metrics on the BAM files.
```

Comment:
All of the workflows are "optional", so I don't think we need to explicitly call this one out.